docs/developer/images.md

```diff
@@ -28,13 +28,27 @@ The color values in the default palette for the MakeCode UI are listed here:
 
 ![Default color chart](/static/developer/default-colors.jpg)
 
-### Changing the default palette
+### User palette settings
+
+Users can change the color palette used for their projects in the Asset Editor under the **Asset** tab. Pressing the **Colors** button will display the Color Palette window. Several predefined palettes are available to choose from. The user can create a new "custom" palette if desired and pick the 16 colors to fill it. This palette will be custom to the project and become its default palette.
```

Comment:
```suggestion
Users can change the color palette used for their projects in the Asset Editor under the **Asset** tab. Pressing the **Colors** button will display the Color Palette window. Several predefined palettes are available to choose from. The user can create a new "custom" palette if desired and pick the 15 colors to fill it. This palette will be custom to the project and can be set as the default palette by clicking **Apply**.
```

Reply:
The first color is not displayed to the user since it is reserved for transparency, so there are 15 colors rather than 16 colors to choose.